src/config/urls.py

```diff
@@ -65,6 +65,11 @@
     urlpatterns += staticfiles_urlpatterns()
     urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
+    # Dev tools
+    urlpatterns += [
+        path("dev-tools/", include("dev_tools.urls", namespace="dev_tools"))
+    ]
+
```

Comment:
Might be worth putting this in a condition so that DEBUG doesn't assume `DEV_TOOLS_ENABLED`

```suggestion
    if settings.DEV_TOOLS_ENABLED:
        # Dev tools
        urlpatterns += [
            path("dev-tools/", include("dev_tools.urls", namespace="dev_tools"))
        ]

```